test/config.ts

```diff
@@ -52,6 +52,7 @@ const config: Config = {
   testDir: path.join(__dirname, "e2e"), // Search for tests in this directory.
   timeout: 60000, // Each test is given 60 seconds.
   retries: 3, // Retry failing tests 2 times
+  workers: 1,
```

Comment:
does this make tests slower?

Reply:
> does this make tests slower?

Yes 😢 Unfortunately, it seems having these tests run in parallel against the same code-server application leads to flakiness (i.e. weird things happen). It's probably the way code-server is built and/or having concurrent connections to the same instance. 

More about workers: 
> Each worker process creates a new environment to run tests. Different environments always run in different workers. By default, Folio reuses the worker as much as it can to make testing faster, but it will create a new worker when retrying tests, after any test failure, to initialize a new environment, or just to speed up test execution if the worker limit is not reached.
— `folio` [docs](https://github.com/microsoft/folio#workers) (playwright-test uses folio under the hood)